root/etc/e-smith/templates/etc/ssh/sshd_config/40ChallengeResponseAuthentication

```diff
@@ -1 +1 @@
-ChallengeResponseAuthentication no
+ChallengeResponseAuthentication yes
```

Comment:
I'm not sure about this change of a default: RHEL changed the default for more secure installations (see also https://blog.tankywoo.com/linux/2013/09/14/ssh-passwordauthentication-vs-challengeresponseauthentication.html)

I think that config should be changed by password auth prop: if `PasswordAuthentication` is `yes`, change ChallengeResponseAuthentication to `yes` too.